packages/main/src/themes/OptionCustom.css

```diff
@@ -0,0 +1,7 @@
+:host {
+	display: flex;
+	align-items: center;
+	height: var(--_ui5_list_item_base_height);
+	width: 100%;
+	padding: 0 1rem;
```

Comment:
Is this padding intended to be the same as in the list option? If yes, can we reuse variable if exists?